api/google_analytics_provider.py

```diff
@@ -21,8 +21,8 @@ class GoogleAnalyticsProvider(object):
         + "<p>Create a <a href='https://analytics.google.com/analytics/web/provision/?authuser=0#/provision' "
         + "rel='noopener' rel='noreferer' target='_blank'>Google Analytics</a> account, "
         + "or sign into your existing one.</p>"
-        + "<p>To capture data from the Library Simplified Circulation Manager in your Google Analytics account, "
-        + "you must set up a property in Google Analytics for Library Simplified.  In your Google Analytics "
+        + "<p>To capture data from the Palace Collection Circulation Manager in your Google Analytics account, "
```

Comment:
Extra `circulation` slipped in here.

Palace Collection Circulation Manager -> Palace Collection Manager